StatBot/Cards/ActionButton.cs

```diff
@@ -4,91 +4,159 @@
 using StatBot.Interfaces;
 using StatBot.Models;
 using StatBot.Database.PostgresRepositories;
+using StatBot.Commands;
 namespace StatBot.Cards
 {
-    public class ActionButton : ICard
+    public class ActionButton
     {
-        public List<string> ButtonsName { get; set;  }
         public CardAction Action { get; set; }
 
+        public struct ButtonSettings
+        {
+            private int _actionType;
+            private string _buttonTittle;
+            private bool _isStatistic;
+            private int _statistic_Id;
+
+            public int ActionType { get => _actionType; set => _actionType = value; }
+            public int Statistic_Id { get => _statistic_Id; set => _statistic_Id = value; }
+            public string ButtonTittle { get => _buttonTittle; set => _buttonTittle = value; }
+            public bool IsStatistic { get => _isStatistic; set => _isStatistic = value; }
+            
+            public ButtonSettings(int Type, int Id, string Tittle, bool IsStat)
+            {
+                _actionType = Type;
+                _statistic_Id = Id;
+                _buttonTittle = Tittle;
+                _isStatistic = IsStat;
+            }
+        }
+        public ButtonSettings buttonSettings;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this field 'private' and encapsulate it in a 'public' property. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://172.21.13.172:9091/coding_rules#rule_key=csharpsquid%3AS1104)